libft/sources/strings/ft_split_str.c

```diff
@@ -38,23 +21,25 @@ static void	calc_tokens(const char *pp, int *token_q, t_delims *ar)
 	p = pp;
 	while (*p)
 	{
-		while (*p == ' ')
+		while (*p == ' ' || *p == '\t' || *p == '\n')
```

Comment:
looks dirty. what is this? are these part of delims?

Reply:
delims here are operators, i split first by operators. 
ok, i deleted this '\t' and '\n' because i handle this in second step splitting ft_split_skip_delim and i dont want now to pass second structure here.